pkg/controller/batchrelease/workloads/cloneset_control_plane.go

```diff
@@ -56,34 +57,34 @@ func NewCloneSetRolloutController(cli client.Client, recorder record.EventRecord
 
 // VerifyWorkload verifies that the workload is ready to execute release plan
 func (c *CloneSetRolloutController) VerifyWorkload() (bool, error) {
-	var err error
+	var message string
 	defer func() {
-		if err != nil {
-			klog.Warningf(err.Error())
-			c.recorder.Event(c.parentController, v1.EventTypeWarning, "VerifyFailed", err.Error())
+		if message != "" {
+			klog.Warningf(message)
```

Comment:
1. no need to warning log explicitly, recorder event function will also log the message
2. if error is nil, do we still need to emit VerfiyFailed message, such as "CloneSet(%v) is still reconciling, wait for it to be don" ?